app/assets/javascripts/discourse/lib/load-script.js.es6

```diff
@@ -100,7 +101,16 @@ export default function loadScript(url, opts) {
       ajax({
         url: cdnUrl,
         dataType: opts.css ? "text" : "script",
-        cache: true
+        cache: true,
+        converters: {
```

Comment:
This is necessary for async js loading to work with a relatively restrictive CSP. The code is borrowed almost verbatim from:

https://github.com/discourse/discourse/blob/420739e12c7fbf492f914d2c18ce73d501712c98/vendor/assets/javascripts/jquery.js#L97-L112

(with the inclusion of a nonce attribute)